### PR TITLE
[CLOSES #432] Add resource fallback to ZiplineLoader

### LINKED ISSUE #432: Add load from Resources before load from cache/network in ZiplineLoader

### LINKED ISSUE #433: Find native specific SqlException class to catch before iOS usage
https://github.com/cashapp/zipline/pull/429#discussion_r813071552

### PR DESCRIPTION
**Pull request by adrw**
[CLOSES #432] Add resource fallback to ZiplineLoader

### CODE REVIEW

File: zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt [hunk 1]
```diff
@@ -65,6 +67,7 @@ class ZiplineLoader(
   )
 
   suspend fun load(zipline: Zipline, url: String) {
+    // TODO fallback to manifest shipped in resources for offline support
```
Comment (swankjesse) [inline zipline-loader/src/commonMain/kotlin/app/cash/zipline/loader/ZiplineLoader.kt:70]: Nice.

**Review by swankjesse — APPROVED** [review]

**Comment by JakeWharton** [comment]
This does not appear to address #433.

**Comment by adrw** [comment]
@JakeWharton apologies, I meant to close #432

**Comment by JakeWharton** [comment]
Whoops yep I figured that after poking around. I did reopen #433 but did not close #432.

**Pull request merged**